src/functionloading.jl

```diff
@@ -19,7 +19,7 @@ function debug_opengl_expr(func_name, args)
             err = glGetError()
             if err != GL_NO_ERROR
                 arguments = gl_represent.(tuple($(args...)))
-                @warn("OpenGL call to $($func_name), with arguments: $(arguments)
+                error("OpenGL call to $($func_name), with arguments: $(arguments)
```

Comment:
```suggestion
                @error("OpenGL call to $($func_name), with arguments: $(arguments)
```

you need the macro for the stacktrace, I think

Reply:
No this just interrupts

Reply:
So you get an error & stacktrace

Reply:
perfect, looks like it worked